### PR TITLE
Rename "Control" key to "Ctrl" and add "_pressed" suffix to all InputEventWithModifiers properties/methods

### PR DESCRIPTION
**Pull request by AaronRecord**
See https://github.com/godotengine/godot/issues/16863#issuecomment-821513317
```cpp
if (event->get_control()) {
...
}
```
becomes
```cpp
if (event->is_ctrl_pressed()) {
...
}
```
Renames:
- `KEY_CONTROL` to `KEY_CTRL`
- `MOD_CONTROL` to `MOD_CTRL`
- `InputEventWithModifiers`:
  * Affixed `_pressed` to all modifiers
  * `control` to `ctrl`(`_pressed`)
  * renamed `meta`(`_pressed`)'s setters and getters to match the property name (`[...]_metakey_pressed()` -> `[...]_meta_pressed()`)
  * `get_` to `is_` for modifiers' getters

On MacBook keyboards, <kbd>Ctrl</kbd> is often <kbd>control</kbd>, but I believe <kbd>Ctrl</kbd> is more common ([the documentation](https://docs.godotengine.org/en/latest/classes/class_inputeventwithmodifiers.html#class-inputeventwithmodifiers-property-control) literally calls it the <kbd>Ctrl</kbd> modifier, and most comments in the code say <kbd>Ctrl</kbd>) and more importantly won't get confused with [`Control`](https://docs.godotengine.org/en/stable/classes/class_control.html) nodes.

### CODE REVIEW

File: core/os/keyboard.cpp [hunk 1]
```diff
@@ -60,7 +60,7 @@ static const _KeyCodeText _keycodes[] = {
 		{KEY_PAGEUP                        ,"PageUp"},
 		{KEY_PAGEDOWN                      ,"PageDown"},
 		{KEY_SHIFT                         ,"Shift"},
-		{KEY_CONTROL                       ,"Control"},
+		{KEY_CTRL                       ,"Ctrl"},
 #ifdef OSX_ENABLED
```
Comment (madmiraal) [inline core/os/keyboard.cpp:64]: Looks like the alignment needs fixing.

**Review by madmiraal — APPROVED** [review]
Looks good to me. Just one small nitpick below.

File: core/os/keyboard.cpp (hunk 1 above)
Comment (LightningAA) [reply core/os/keyboard.cpp:64]: Should be fixed now

**Comment by groud** [comment]
I'd says the change make sense, though I am worried about the fact it might  impact discoverability, as users may not search for "ctrl" when looking for something related to the "control" key. But generally I would say it's an improvement as it avoids the confusion with Control nodes, so I am ok with the change.

**Comment by Faless** [comment]
It's worth mentioning the key is indeed called `control`, of which `ctrl` is just an abbreviation.
I don't mind the change, I just don't want it to become another `trash`/`recycle bin`/`delete` deal.

**Comment by Faless** [comment]
> as users may not search for "ctrl" when looking for something related to the "control" key.

For discoverability we could alias:
`KEY_CTRL = KEY_CONTROL`
`KEY_ALT = KEY_ALTERNATE`

etc to match ISO definitions: https://en.wikipedia.org/wiki/ISO/IEC_9995#Function_keys

**Comment by LightningAA** [comment]
> KEY_ALT = KEY_ALTERNATE

I'm not sure I've literally ever heard anyone call the <kbd>alt</kbd> key the <kbd>alternate</kbd> key

**Comment by Faless** [comment]
> I'm not sure I've literally ever heard anyone call the alt key the alternate key

The ISO standard does?

**Comment by LightningAA** [comment]
> The ISO standard does?

True, but if no one calls it that does it matter?

**Comment by Faless** [comment]
Well "the standard" is clearly different from "no one".
And yes, following standards do matter.

But as I said before, I don't really care, it was just a suggestion.
Clearly, no one wanted that suggestion, so I'm going to unfollow this.

**Comment by Riteo** [comment]
I'm quite okay with this, but why didn't you rename `metakey` wherever it already were in order to make it consistent with `meta`, instead of doing the opposite? IMO it would be more consistent with the naming scheme to have it just as `meta`.

**Comment by LightningAA** [comment]
> I'm quite okay with this, but why didn't you rename metakey wherever it already were in order to make it consistent with meta, instead of doing the opposite? IMO it would be more consistent with the naming scheme to have it just as meta.

Fair point, I changed it back to <kbd>meta</kbd>

**Comment by akien-mga** [comment]
It would have been better for review purposes to split those changes into two PRs, the rename of `Control` to `Ctrl` is just one aspect of this PR which changes a lot of the `InputEventWithModifiers` API.

That being said, now that the work is done let's keep it this way.

I don't have a strong opinion on rename Control to Ctrl, but I'd just raise a comment that the name of keys doesn't necessarily match what's printed on the key caps... My laptop has "pg up" / "pg dn", yet we have `KEY_PAGEUP`, `KEY_PAGEDOWN`. A German keyboard has "Strg" for the control key. We also have `KEY_ESCAPE` when most keycaps have "esc" or "Esc" printed on them.

I do not think that we should rename `KEY_ESCAPE` so I just want to make it clear that the aim is not to have a 1:1 correspondence between key names and what's printed on key caps. Following a standard might be a better way to decide what should be used.

Now, if most are in favor of the rename I don't mind either way.

-----

On the other proposed changes to `InputEventWithModifiers`, I haven't seen much discussion. I'm not sure it's worth it to add a `_pressed` suffix to *all* properties, isn't it clearly inferred by the description of `InputEventWithModifiers`? It's an `InputEvent` and it has modifiers, which can be Alt, Shift, Ctrl, etc.

**Comment by LightningAA** [comment]
> On the other proposed changes to InputEventWithModifiers, I haven't seen much discussion. I'm not sure it's worth it to add a _pressed suffix to all properties, isn't it clearly inferred by the description of InputEventWithModifiers? It's an InputEvent and it has modifiers, which can be Alt, Shift, Ctrl, etc.

My reasoning for this change was mostly because I thought adding `_pressed` would help the code read easier. 

For example:
```cpp
if (event->is_meta() || event->is_command()) {
    // ...
}
```
makes less sense to me than:
```cpp
if (event->is_meta_pressed() || event->is_command_pressed()) {
    // ...
}
```
The first one sounds like it's asking if the event is meta or if the event is a command (whatever that'd mean), which might be a little confusing to someone new to the codebase/unfamiliar with the `InputEventWithModifiers` class, the `_pressed` makes it clear it's referring to the meta/command keys.

**Review by akien-mga — APPROVED** [review]

**Comment by akien-mga** [comment]
Thanks!

**Pull request merged**